mwviews/api/pageviews.py

```diff
@@ -301,6 +303,120 @@ def top_articles(
             'The pageview API returned nothing useful at: {}'.format(url)
         )
 
+    def top_articles_by_country(
```

Comment:
`top_by_country` here to match the above

Reply:
fixed